examples/js/loaders/TDSLoader.js

```diff
@@ -43,9 +43,13 @@ THREE.TDSLoader.prototype = {
 
 		var scope = this;
 
-		var loader = new THREE.FileLoader( this.manager );
+		var path = scope.path === undefined ? THREE.Loader.prototype.extractUrlBase( url ) : scope.path;

```

Comment:
I would change this for a if condition and only dedefine the path "load.setPath" if there was no path defined.

Also,  path is never undefined it starts defined as "" to avoid error on string concatenation.

Reply:
@cnspaha The implementation of `TDSLoader` varies from `GLTFLoader` and `ColladaLoader`. So we can't do just a copy/paste of the code unless we change parts of `TDSLoader`. But i don't think this is necessary.

@tentone Since you are the original author, i'll leave it up to you to guide @cnspaha through the PR 😊

Reply:
@Mugen87 
yeah, you are right.

@tentone 
since you know the good/functionality of the TDSLoader better than me, would you mind to create a PR with the correct code, regarding the issues from this PR?

Reply:
@cnspaha 

I think we can keep this PR, the file seems to be good i would only change the load method to this

```
	load: function ( url, onLoad, onProgress, onError ) {
	
		var scope = this;

		if ( this.path === undefined )	{

			this.path = THREE.Loader.prototype.extractUrlBase( url );

		}
		
		var loader = new THREE.FileLoader( this.manager );

		loader.setResponseType( 'arraybuffer' );
		
		loader.setPath( this.path );

		loader.load( url, function ( data ) {

			onLoad( scope.parse( data ) );

		}, onProgress, onError );

	},
```

Reply:
@tentone 
but as you said yourself ```this.path``` is always defined with an empty string, so
```this.path = THREE.Loader.prototype.extractUrlBase( url );``` is never reached? So the whole if branch is useless?

Reply:
Oh my bad, youre right missed that, it shoud be this instead

if ( this.path === "" ) {
	this.path = THREE.Loader.prototype.extractUrlBase( url );
}

Reply:
yeah, sounds reasonable to me ;)
will wait for @Mugen87 s opinion about your load method and if hes also fine with it I will commit it ;)

Reply:
Go for it 👍

Reply:
done :)